platform/frontends/x2cpg/src/main/scala/io/appthreat/x2cpg/passes/frontend/XTypeRecovery.scala

```
@@ -71,8 +71,11 @@ abstract class XTypeRecoveryPass[CompilationUnitType <: AstNode](
         val state     = XTypeRecoveryState(config, stopEarly = stopEarly)
         try
           Iterator.from(0).takeWhile(_ < config.iterations).foreach { i =>
-            val newState = state.copy(currentIteration = i)
-            generateRecoveryPass(newState).createAndApply()
+              try
+                val newState = state.copy(currentIteration = i)
+                generateRecoveryPass(newState).createAndApply()
+              catch
+                case _: Exception =>
```

This suppresses errors such as this. Maybe, we can fix the underlying issue better sometime or replace this pass altogether.

https://github.com/AppThreat/atom/issues/153